docs/source/plugins.rst

```diff
@@ -311,3 +312,14 @@ The following known plugins provide support for running `MLflow projects <https:
 against custom execution backends.
 
 - `mlflow-yarn <https://github.com/criteo/mlflow-yarn>`_ Running mlflow on Hadoop/YARN
+
```

Comment:
This is an upstream change, which is not available in the current repo. It will not been shown as a new addition, when raising an upstream PR